modules/core-updates/8.1.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Changes in PrestaShop 8.1
+menuTitle: Changes in 8.1
+---
+
+<style>
+/* condensed lists in this article */
+#body-inner li, #body-inner li ul, li p { margin-bottom: 0.2rem}
+/* deprecation indicators */
+#body-inner depre {font-size: 85%; color: #666; font-style: italic; vertical-align: middle }
+#body-inner depre::before {content: ' – '}
+</style>
+
+# Notable changes in PrestaShop 8.1
+
+## PHP support
+
+PrestaShop 8.1 has same PHP requirement as Prestashop 8.0.
```

Comment:
i find it interesting to keep it for a quick reference, but this is not needed, opinions?

Reply:
I like this idea as well